pkg/sql/copy_file_upload.go

```diff
@@ -39,9 +39,10 @@ var copyFileOptionExpectValues = map[string]KVStringOptValidate{
 var _ copyMachineInterface = &fileUploadMachine{}
 
 type fileUploadMachine struct {
-	c           *copyMachine
-	writeToFile *io.PipeWriter
-	writeError  chan error
+	c              *copyMachine
+	writeToFile    *io.PipeWriter
+	done           chan bool
```

Comment:
this can be a simple `sync.WaitGroup` at this point since it's not exchanging any information over the chan, and is just a synchronization point.